main_app/views.py

```diff
@@ -157,14 +157,18 @@ def items_details(request, item_id):
             quantity_log.item = item
             quantity_log.user = request.user
             quantity_log.previous_count = item.quantity_current
-            item.quantity_current += quantity_log.change
+            item.quantity_current = quantity_log.change
```

Comment:
Had to change this line here as it was the main reason for the item.quantity_current to not show the same number that was input into the change form.